src/views/Article.vue

```diff
@@ -3,51 +3,75 @@
         <Back></Back>
         <div class="article" @mouseup="onAfterSelect" @touchend="onAfterSelect" v-html="currentArticle"></div>
         <div :style="selectionToolbarStyle" class="toolbar">
-            <div @click="onUseText">Use this text</div>
-            <div @click="onDismiss">Dismiss</div>
+            <div @click="onUseText">Highlight</div>
+            <div @click="onDismiss">Clear</div>
         </div>
+        <div class="overlay" v-if="showImages"></div>
+        <ArticleImages :images="articleImages" :onSubmit="onUseImage" v-if="showImages" class="images" />
     </div>
 </template>
 <script>
   import { mapGetters, mapActions } from 'vuex'
   import Back from '@components/Back.vue'
+  import ArticleImages from '@components/ArticleImages.vue'
 
   export default {
     name: 'Article',
     props: ['article'],
-    components: { Back },
+    components: { Back, ArticleImages },
     data: () => {
       return {
         selectionToolbarStyle: {
           display: 'none',
           position: 'absolute'
         },
-        selectedText: null
+        selectedText: null,
+        showImages: false,
+        images: []
+      }
+    },
+    computed: {
+      ...mapGetters( ['currentArticle'] ),
+      articleImages: () => {
+        return Array.from(document.querySelector('.article').querySelectorAll(('img'))).map(img => {
```

Comment:
learning from another project, capture all img will always get some unexpected images, for example in `Catherine,%20Duchess%20of%20Cambridge`
<img src="https://user-images.githubusercontent.com/2560096/135518589-e482ffd1-76df-41c6-a68d-5da8af5d1614.png" width="300"/>

Reply:
Yeah I saw that, I expect the media-list endpoint to be more accurate but for the prototype I thought this was acceptable. Will improve it if needed.

Reply:
make sense to me 👍